### PR TITLE
Add builtin rsqrt function

### PR DESCRIPTION
**Pull request by AndreiPotapov**
We have fast intrinsics for 1/sqrt and it's estimate version.

### CODE REVIEW

**Comment by AntonYudintsev** [comment]
It seems rsqrt is better name, than invsqrt (HLSL, GLSL)

File: include/daScript/simulate/sim_policy.h [hunk 1]
```diff
@@ -147,16 +147,18 @@ namespace  das {
...
-        static __forceinline float Floor ( float a, Context & )          { return v_extract_x(v_floor(v_splats(a))); }
-        static __forceinline float Ceil  ( float a, Context & )          { return v_extract_x(v_ceil(v_splats(a))); }
-        static __forceinline float Sqrt  ( float a, Context & )          { return v_extract_x(v_sqrt_x(v_splats(a))); }
-        static __forceinline float Min   ( float a, float b, Context & ) { return a < b ? a : b; }
-        static __forceinline float Max   ( float a, float b, Context & ) { return a > b ? a : b; }
-        static __forceinline float Sat   ( float a, Context & )          { return a < 0 ? 0 : (a > 1 ? 1 : a); }
-        static __forceinline float Mad   ( float a, float b, float c, Context & ) { return a*b + c; }
-        static __forceinline float Lerp  ( float a, float b, float t, Context & ) { return (b-a)*t +a; }
-        static __forceinline float Clamp  ( float t, float a, float b, Context & ){ return t>a ? (t<b ? t : b) : a; }
+        static __forceinline float Abs      ( float a, Context & )          { return v_extract_x(v_abs(v_splats(a))); }
```
Comment (AndreiPotapov) [inline include/daScript/simulate/sim_policy.h]: btw we don't need splats there

Comment (AntonYudintsev) [reply include/daScript/simulate/sim_policy.h]: well, we do

Comment (AntonYudintsev) [reply include/daScript/simulate/sim_policy.h]: it is float, and v_abs takes vec4f

Comment (AndreiPotapov) [reply include/daScript/simulate/sim_policy.h]: than we extracting only one component. nan's is not problem for simd instructions.

Comment (AntonYudintsev) [reply include/daScript/simulate/sim_policy.h]: yep
so on x86 (32 bit) it is different regs, and so the only way to do it

**Pull request merged**